app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -76,21 +75,30 @@ class RegisterController @Inject()(val messagesApi: MessagesApi,
                   Logger.error(s"Could not perform eligibility check: $error")
                   InternalServerError("")
                 }, _.result.fold(
-                  SeeOther(routes.RegisterController.notEligible().url))(
-                  userDetails ⇒ Ok(views.html.register.confirm_details(userDetails)))
+                  missingInfos ⇒ {
+                    Logger.error(s"user ${maybeNino.get} has missing information: ${missingInfos.missingInfo.mkString(",")}")
```

Comment:
`.get` will throw if the `Option` is `None` - it's safer to `yield` the nino from the for-comprehension above and bring it into scope here